dask/tests/test_utils.py

```diff
@@ -54,11 +54,31 @@ class Bar(object):
         pass
     b = Bar()
     assert foo(1) == 2
+    assert foo.dispatch(int)(1) == 2
     assert foo(1.0) == 0.0
     assert foo(b) == b
     assert foo((1, 2.0, b)) == (2, 1.0, b)
 
 
+def test_dispatch_lazy():
+    # this tests the recursive component of dispatch
+    foo = Dispatch()
+    foo.register(int, lambda a: a)
+
+    import numpy as np
```

Comment:
We don't require `numpy` for this test file. You'd either need to use `pytest.importorskip('numpy')` here, or (better) use a module from the standard library. Perhaps `decimal`?